collect_app/src/main/java/org/odk/collect/android/utilities/Appearances.kt

```diff
@@ -138,10 +134,10 @@ object Appearances {
                         }
                     }
                 } catch (e: Exception) {
-                    Timber.e(Error(EXCEPTION_PARSING_COLUMNS + columnsAppearance))
+                    // ignore
```

Comment:
While here, could you look to simplify this a bit? There's no point to the inner try/catch, right?

Reply:
Rights, done.